// faker/lib/locales.js
exports['az'] = require('faker/lib/locales/az');
exports['ar'] = require('faker/lib/locales/ar');
exports['cz'] = require('faker/lib/locales/cz');
exports['de'] = require('faker/lib/locales/de');
exports['de_AT'] = require('faker/lib/locales/de_AT');
exports['de_CH'] = require('faker/lib/locales/de_CH');
exports['en'] = require('faker/lib/locales/en');
exports['en_AU'] = require('faker/lib/locales/en_AU');
exports['en_AU_ocker'] = require('faker/lib/locales/en_AU_ocker');
exports['en_BORK'] = require('faker/lib/locales/en_BORK');
exports['en_CA'] = require('faker/lib/locales/en_CA');
exports['en_GB'] = require('faker/lib/locales/en_GB');
exports['en_IE'] = require('faker/lib/locales/en_IE');
exports['en_IND'] = require('faker/lib/locales/en_IND');
exports['en_US'] = require('faker/lib/locales/en_US');
exports['en_ZA'] = require('faker/lib/locales/en_ZA');
exports['es'] = require('faker/lib/locales/es');
exports['es_MX'] = require('faker/lib/locales/es_MX');
exports['he'] = require('faker/lib/locales/he');
exports['fa'] = require('faker/lib/locales/fa');
exports['fi'] = require('faker/lib/locales/fi');
exports['fr'] = require('faker/lib/locales/fr');
exports['fr_CA'] = require('faker/lib/locales/fr_CA');
exports['fr_CH'] = require('faker/lib/locales/fr_CH');
exports['ge'] = require('faker/lib/locales/ge');
exports['hy'] = require('faker/lib/locales/hy');
exports['hr'] = require('faker/lib/locales/hr');
exports['id_ID'] = require('faker/lib/locales/id_ID');
exports['it'] = require('faker/lib/locales/it');
exports['ja'] = require('faker/lib/locales/ja');
exports['ko'] = require('faker/lib/locales/ko');
exports['nb_NO'] = require('faker/lib/locales/nb_NO');
exports['ne'] = require('faker/lib/locales/ne');
exports['nl'] = require('faker/lib/locales/nl');
exports['nl_BE'] = require('faker/lib/locales/nl_BE');
exports['pl'] = require('faker/lib/locales/pl');
exports['pt_BR'] = require('faker/lib/locales/pt_BR');
exports['pt_PT'] = require('faker/lib/locales/pt_PT');
exports['ro'] = require('faker/lib/locales/ro');
exports['ru'] = require('faker/lib/locales/ru');
exports['sk'] = require('faker/lib/locales/sk');
exports['sv'] = require('faker/lib/locales/sv');
exports['tr'] = require('faker/lib/locales/tr');
exports['uk'] = require('faker/lib/locales/uk');
exports['vi'] = require('faker/lib/locales/vi');
exports['zh_CN'] = require('faker/lib/locales/zh_CN');
exports['zh_TW'] = require('faker/lib/locales/zh_TW');
